EIPS/eip-token-standard.md

```diff
@@ -0,0 +1,153 @@
+## Preamble
+
+    EIP: <to be assigned> (I Suggest 20)
+    Title: <EIP title>
+    Author: Fabian Vogelsteller <fabian@ethereum.org>, Vitalik Buterin <vitalik.buterin@ethereum.org>
+    Type: Informational
+    Status: Draft
+    Created: 2015-11-19
+
+
+## Simple Summary
+
+A standard interface for tokens.
+
+
+## Abstract
+
+The following standard allows for the implementation of a standard API for tokens within their smart contracts:
```

Comment:
The following standard allows for the implementation of a standard API for tokens within their smart contracts.
This standard provides basic functionality to transfer tokens and allows them to be approved to be spent by another on-chain third party.